db/migrate/20170209160355_add_branch_to_pull_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddBranchToPullRequests < ActiveRecord::Migration[5.0]
+  def change
+    add_column :pull_requests, :branch, :string, null: true
```

Comment:
Not really. That feature being alpha, we're the sole users of it, and only used it on a test stask that I will delete.

Also I checked if you try to delete the `nil` branch, it behave like the branch doesn't exist so it's handled fine.